website/docs/r/eip.html.markdown

```diff
@@ -112,6 +122,7 @@ In addition to all arguments above, the following attributes are exported:
 * `public_ip` - Contains the public IP address.
 * `instance` - Contains the ID of the attached instance.
 * `network_interface` - Contains the ID of the attached network interface.
+* `public_ipv4_pool` - IPv4 address pool.
```

Comment:
This line should be removed due to the `Computed` -> `Default` change. 👍

Reply:
As I mentioned above EC2-Classic does not allow to use public_ipv4_pool attribute.